docs/api.yml

```diff
@@ -98,6 +98,7 @@ sidebar:
       - "api-reference/expressions/not"
       - "api-reference/expressions/typeof"
       - "api-reference/expressions/when"
+      - "api-reference/expressions/now"
```

Comment:
We keep all API reference tabs alphabetically sorted within a section. Please move it after `not`

Reply:
changed